swap/tests/harness/mod.rs

```diff
@@ -675,36 +683,15 @@ impl TestContext {
     }
 
     pub async fn assert_bob_refunded(&self, state: BobState) {
-        self.bob_internal_bitcoin_wallet.sync().await.unwrap();
-
-        let lock_tx_id = if let BobState::BtcRefunded(state4) = state {
-            state4.tx_lock_id()
-        } else {
-            panic!("Bob in not in btc refunded state: {:?}", state);
-        };
-        let lock_tx_bitcoin_fee = self
-            .bob_internal_bitcoin_wallet
-            .transaction_fee(lock_tx_id)
-            .await
-            .unwrap();
+        assert!(matches!(state, BobState::BtcRefunded(..)));
 
-        let btc_balance_after_swap = self.bob_internal_bitcoin_wallet.balance().await.unwrap();
-
-        let cancel_fee = self
-            .alice_internal_bitcoin_wallet
-            .estimate_fee(TxCancel::weight(), self.btc_amount)
-            .await
-            .expect("To estimate fee correctly");
-        let refund_fee = self
-            .alice_internal_bitcoin_wallet
-            .estimate_fee(TxRefund::weight(), self.btc_amount)
-            .await
-            .expect("To estimate fee correctly");
-
-        let bob_cancelled_and_refunded = btc_balance_after_swap
-            == self.bob_starting_balances.btc - lock_tx_bitcoin_fee - cancel_fee - refund_fee;
-
-        assert!(bob_cancelled_and_refunded);
```

Comment:
Same thing here, please explain why it is ok to delete this. Ideally in a dedicated refactoring commit that doesn't involve other functionality.